### DISCUSSION TITLE
Github Pages Build Error

### DISCUSSION THREAD

**Discussion by DexterCodes5** [community/community#55509] [Programming Help]
### Body

I'm trying to deploy a vite react app to Github Pages. In the build I see this error: 
Could not resolve "../../../icons/payPalIcon.svg" from "src/layouts/CheckoutLayout/PaymentPage/PaymentPage.jsx"
file: /home/runner/work/fixerpro-frontend/fixerpro-frontend/src/layouts/CheckoutLayout/PaymentPage/PaymentPage.jsx
error during build:

When I run the vite app on my machine with "npm run dev", it runs fine and no such error. I don't know what to do.

This is the repo:
https://github.com/DexterCodes5/fixerpro-frontend

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by SRayen** [comment]
i have tried to build your project with the cmd 'npm run build' and it worked for me without errors!
![Capture d’écran 2023-05-16 224935](https://github.com/community/community/assets/13922445/b5429d7c-527c-4526-857a-1583da29c4fd)

**Reply by jge162** [reply]
You need to fix the pathing for the error message you are getting. GitHub pages cannot find the svg file. Trace it and fix pathing. Even though it works on your remote environment GitHub pages cannot find the svg file. 

`Vite is having trouble resolving the path to the SVG file ("../../../icons/payPalIcon.svg") from the file PaymentPage.jsx in your project.`

**Reply by jge162** [reply]
the .svg file is not even in your icon section look. 
<img width="1198" alt="Screenshot 2023-05-16 at 2 55 56 PM" src="https://github.com/community/community/assets/31228460/a8e80637-3c08-4556-8045-350f49216002">

**Reply by DexterCodes5** [reply]
I didn't notice that. Seems Github Pages did not like my svg.

**Comment by SRayen** [comment]
Try to update "../../../icons/payPalIcon.svg" to "../../../icons/payPalIcon.png"

**Reply by DexterCodes5** [reply]
I fixed it, now I have new problems : (

**Reply by SRayen** [reply]
Make the discussion as Answered at first!
Then tell us your new problems.
